nondex-maven-plugin/src/it/clean-it/pom.xml

```diff
@@ -23,10 +23,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.19.1</version>
-        <configuration>
-          <excludedGroups>edu.illinois.nondex.it.OtherIgnore</excludedGroups>
-        </configuration>
+        <version>2.20</version>
```

Comment:
Make a parent `pom.xml` that each IT's `pom.xml` file can inherit from, so that you can make version changes in only one place.